deku-c/client/src/deku-c/index.ts

```diff
@@ -17,7 +13,6 @@ export type Settings = {
 export class DekuCClient extends DekuPClient {
   readonly ligoRpc?: string;
```

Comment:
maybe change name?

Reply:
/shrug. Can't think of a better one. Gonna merge with this for now.